bayesian/api/api_v2.py

```diff
@@ -303,6 +305,34 @@ def stack_analyses():
         raise HTTPError(500, e.args[0])
 
 
+@api_v2.route('/stack-report/<stack_id>', methods=['GET'])
+def stack_report_url(stack_id: str):
```

Comment:
Didn't we decided to do the RDS access on SA GET request when UUID is not present?

Reply:
No, Prashant and I re-thought that and concluded on "RDS call in the Redirect URL" Approach would be a cleaner one.

Reply:
The auth decorator expects a UUID , it is better to be consistent than to add hacks to work around that.

Reply:
@prashbnair , the existing decorator can be kept as is. Without uuid, it just skips RDS call and sets the user type as unregistered. All we need to do is add one more check(the same one which @deepak1725 had put into the redir endpoint) in the SA GET which changes the user registration status after obtaining the UUID. It would be simple too and RDS access will be under 3scale which was raised as a concern during our last discussion.

Reply:
@arajkumar Stack Report UI doesn't accept empty `uuid/crda_key`

Reply:
@arajkumar It does not make for the user registration status to handled in different places, the auth filter does it.  That was the main reason for doing it this way. 
We need to ensure that there cannot be any SQL injection and we should be fine.